src/one_sdk.ts

```diff
@@ -72,7 +73,10 @@ export function createResolver(
       }
     } catch (err) {
       debug('Perform exception', err);
-      throw err;
+      // This is needed because OneSDK v1.x throws errors which don't inherit from Error,
+      // causing graphql-js to throw the original error away.
+      // While we do mapping, we can also extract SDK-specific properties to GraphQL extensions
+      throw remapSdkError(err as OneSdkError);
```

Comment:
suggestion: assert type before passing instead of casting.

```suggestion
      if (isOneSdkError(err)) {
        throw remapOneSdkError(err);
      }
      
      throw err;    
```